templates/terraform/schema_property.erb

```diff
@@ -86,6 +90,15 @@
         <% end -%>
       },
   <% end -%>
+  <% if property.is_set -%>
+    <% if !property.set_hash_func.nil? -%>
+    Set: <%= property.set_hash_func -%>,
+    <% elsif property.item_type.is_a?(String) -%>
+    Set: schema.HashString,
```

Comment:
Need to confirm for sure, but I think this is automatically the default for sets of strings